src/g_game.c

```diff
@@ -1620,8 +1620,17 @@ void G_DoLoadLevel(boolean resetplayer)
 
 	levelstarttic = gametic; // for time calculation
 
-	if (wipegamestate == GS_LEVEL)
-		wipegamestate = -1; // force a wipe
+    // miru: postlevelwipe - just in case
+    if (!mapheaderinfo[gamemap-1]->postlevelwipe)
+    {
+        if (wipegamestate == GS_LEVEL)
+            wipegamestate = -1; // force a wipe
+    }
+    else
+    {
+        if (wipegamestate == GS_LEVEL)
+            wipegamestate = -1; // force a wipe
+    }
```

Comment:
Both sides of this if statement accomplish the same thing, so you could just leave it out